js/src/index.ts

```diff
@@ -38,12 +38,14 @@ export class VolumeWidgetView extends widgets.DOMWidgetView {
           rawData: volume,
           rawDims: dimensions,
           viewerChannelSettings: {
+            maskChannelName: "",
```

Comment:
i cant' remember if we have a gh issue for resolving this, but we should :)

Reply:
I'd definitely heard you mention it before, but hadn't understood what you meant until I had this error. A fix for this come with the release that knocks the -dev out of the viewer version.